Условные операторы 2/Условные операторы 2/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Условные_операторы_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            while (true)
+            {
+                    Console.WriteLine("Введите символ");
+                char x = char.Parse(Console.ReadLine());
```

Comment:
Если будет введена строка, то будет ошибка выполнения, потому что строку нельзя конвертировать в символ. Можно в этом случае обрабатывать первый символ введенной строки
```suggestion
                char x = Console.ReadLine()[0];
```